test/e2e/volumes.go

```diff
@@ -381,9 +381,8 @@ func deleteCinderVolume(name string) error {
 	return err
 }
 
-// These tests need privileged containers, which are disabled by default.  Run
-// the test with "go run hack/e2e.go ... --ginkgo.focus=[Feature:Volumes]"
-var _ = framework.KubeDescribe("Volumes [Feature:Volumes]", func() {
+// These tests need privileged containers, which are disabled by default.
```

Comment:
The common func which creates the server pods for all in-tree plugins (startVolumeServer) sets privileged to true for all server pods, not just the ones I removed `[Feature:Volumes]` from. I have not tried to remove privileged to see if the mounts work. I believe there's a CAP that can be added to pods to obviate the need for privileged, but feel that a security change is outside the scope of this pr. The description of [Feature:], cited in the doc in the description, does not mention privilege containers.